test/fw/ptl/utils/pbs_procutils.py

```diff
@@ -235,20 +241,26 @@ def __init__(self, name=None, pid=None):
         self.rss = None
         self.vsz = None
         self.pcpu = None
+        self.pmem = None
+        self.size = None
+        self.cputime = None
         self.time = time.time()
         self.command = None
 
     def __str__(self):
-        return "%s pid: %s rss: %s vsz: %s pcpu: %s command: %s" % \
+        return "%s pid: %s rss: %s vsz: %s pcpu: %s pmem: %s \
+               size: %s cputime: %s command: %s" % \
```

Comment:
This will insert all of those spaces into the line. Instead, break the string like so:
```Python
         return (f"really long string {value}"
                 f"really long string {value2}")
```